### ISSUE TITLE
Screen curvature color

### ISSUE THREAD

**Issue by rncar** [Swordfish90/cool-retro-term#520]
any way to change the border color when the curvature is enabled ?

**Comment by fyvfy** [comment]
I second this. Need option to have black borders, like a vignetting in photography. So terminal blends in with LCD monitor borders. Not this ugly whitish corners.

**Comment by Lee-Adamson** [comment]
I agree with this.  Just upgraded and oh man that white border ouch.  ;)  It would be nice to have the option to do the screen curvature without rendering the frame at all, too, for those of us that are hiding LCDs inside of worn out CRT glass.
